### ISSUE TITLE
on the training_nli.py of NLI training example

### ISSUE THREAD

**Issue by lukemao** [UKPLab/sentence-transformers#1152]
Hi,

Thanks for providing an example of training SBERT on NLI examples.

I have a question on the example code provided `training_nli.py` 

At line: `train_loss = losses.SoftmaxLoss(model=model, sentence_embedding_dimension=model.get_sentence_embedding_dimension(), ....`

Does this line automatically prepare the input to the softmax operator. 

When I saw 'input' I mean the concatenated vector:

`(u, v, |u-v|)`

Lu.

**Comment by nreimers** [comment]
Yes

**Comment by lukemao** [comment]
Thanks Nils !
